internal/docs/jacc/JACCSpecAssertions.xml

```diff
@@ -104,24 +104,23 @@
 		</chapters>
 	</location-names>
 	<assertions>
-		<assertion required = "true" impl-spec = "false" defined-by = "technology" status = "active" testable = "true">
+		<assertion required = "false" impl-spec = "false" defined-by = "technology" status = "active" testable = "true">
 			<id>2</id>
-			<description>J2EE 1.4 platforms must implement the contract defined by this JSR.</description>
+			<description>Legacy</description>
```

Comment:
Should all legacy EE references that are less than 8, be updated to "Legacy"?  Or is it better to remove the reference to which EE version introduced the requirement?

Reply:
Here, in the spec document itself, there was a split between requirements for a J2EE 1.4 platform and a J2EE 1.3 platform.

The J2EE 1.3 platform reference was removed entirely, as Jakarta EE obviously has nothing to see about what a J2EE 1.3 container should do.

What was left at that point was "J2EE 1.4 platforms must implement the contract defined by this JSR.", which I could update to "Jakarta EE 9 platforms must implement the contract defined by this specification.", but I think those would be just empty words. With Jakarta Authorization being a part of Jakarta EE 9, that is already implied.

Since I don't know if something will break if assertion IDs have gaps, I opted to just set it to "Legacy".

Reply:
+1 and thanks for explaining!